ovos_gui_plugin_shell_companion/brightness.py

```diff
@@ -75,11 +75,23 @@ def is_auto_dim_enabled(self, message=None):
 
     # Discover the brightness control device interface (HDMI / DSI) on the Raspberry PI
     def discover(self):
+        try:
+            vcgen_path = (subprocess.run(["which", "vcgencmd"], check=True,
+                                         capture_output=True).stdout.decode()
+                          .strip())
+        except subprocess.CalledProcessError:
+            vcgen_path = "/opt/vc/bin/vcgencmd" if (
+                exists("/opt/vc/bin/vcgencmd")) else None
+        if not vcgen_path:
+            LOG.info("Falling back to DSI interface")
+            self.device_interface = "DSI"
         try:
             LOG.info("Discovering brightness control device interface")
-            proc = subprocess.Popen(["/opt/vc/bin/vcgencmd",
-                                     "get_config", "display_default_lcd"], stdout=subprocess.PIPE)
-            if proc.stdout.read().decode("utf-8").strip() == "1":
+            disp = subprocess.run([vcgen_path, "get_config",
+                                   "display_default_lcd"], check=True,
+                                  capture_output=True).stdout.decode().strip()
+            # previous check for '1' return, observed "display_default_lcd=1"
+            if disp in ('1', 'display_default_lcd=1'):
```

Comment:
merged #14 which was older and had the same fix, but this line is not in #14 so can you please rebase?